test/tools/utils.ts

```diff
@@ -100,6 +100,31 @@ export function getEncryptExtraOptions() {
   return {};
 }
 
+export function getKmsProviders(
```

Comment:
Do we want a shared utility for getting kms providers?   Most prose tests seem to specify exactly what to use.

I'm cautious about adding test helpers because we've seen issues with shared helpers in the past (`this.configuration.newClient`, `resolveConnectionString`, ect).

Reply:
It isn't new, it is what all the existing prose tests use, I just moved it to utils. If you look at the top of the tests readme in the specs for FLE it talks about setting these things from the environment.

Reply:
I still think each test should be access the credentials it needs, it's not much work.  Shared utilities for testing have caused us pain in the past.

Reply:
removed